resources/terminology/zib-ContactInformation-TelephoneNumbers-Systems.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="zib-ContactInformation-TelephoneNumbers-systems"/>
+    <url value="http://nictiz.nl/fhir/ValueSet/zib-ContactInformation-TelephoneNumbers-systems"/>
+    <version value="4.0.1"/>
+    <name value="zibContactInformationTelephoneNumbersSystems"/>
+    <title value="zib ContactInformation TelephoneNumbers systems"/>
+    <status value="active"/>
+    <experimental value="false"/>
+    <publisher value="Nictiz"/>
+    <description value="Telecommunications form for contact point, restricted to the use for the TelephoneNumbers concept of zib ContactInformation."/>
+    <immutable value="true"/>
+    <compose>
+        <include>
+            <system value="http://hl7.org/fhir/contact-point-system"/>
+            <concept>
+                <code value="phone"/>
+                <display value="Phone"/>
+            </concept>
+            <concept>
+                <code value="fax"/>
```

Comment:
Moeten de designation er ook niet bij:

<designation>
                    <language value="nl-NL" />
                    <use>
                        <system value="http://snomed.info/sct" />
                        <code value="900000000000013009" />
                        <display value="Synonym" />
                    </use>
                    <value value="Fax" />
                </designation>

Reply:
Geen idee. Wat doet dat?

Reply:
De Nederlandse vertaling van de display geven. Volgens mij worden die nog behoorlijk vaak door PGO's gebruikt voor de vertaling van de concepten.

Reply:
Klinkt leuk. Maar moeten we dat echt doen voor een ValueSet die een bestaande ValueSet inperkt. En moet dat dan ook in het CodeSystem als we dat meeleveren? Of in plaats ervan?

Reply:
Mm, goed punt. Nee, denk het niet. 
Daarnaast komen op de code-specification extensie dan nog de zib codelijsten/valuesets te hangen waar dit al in staat.

Reply:
Nee, dit is de bestaand FHIR core ValueSet die wordt ingeperkt.